### ISSUE TITLE
Create "blocked" label for issues

### ISSUE THREAD

**Issue by uilianries** [conan-io/hooks#326]
Some issues and PR may need to solve another issue from CCI first. They should be marked as "blocked".

**Comment by jgsogo** [comment]
There is already a `blocked` label, although we usually use `infrastructure` for the ones that depend on infra/library.

**Comment by uilianries** [comment]
Nope, we don't have for hooks: https://github.com/conan-io/hooks/labels. 14 labels.

For infrastructure yes, but for cases where we need to discuss first from another repository (e.g. #322). I could block the PR by my review, but first, I don't have admin powers, second, label are useful for filtering, so I could find easier than looking for restricted PR/Issues by review.

**Comment by jgsogo** [comment]
I was thinking about labels in `conan-io/conan-center-index` 😓

**Comment by jgsogo** [comment]
Done: https://github.com/conan-io/hooks/labels/blocked

**Comment by uilianries** [comment]
Gracias!
